library/src/CMakeLists.txt

```diff
@@ -67,7 +67,7 @@ add_library( roc::hipblas ALIAS hipblas )
 set(static_depends)
 
 if( WIN32 )
-  find_package( hipblas-common REQUIRED CONFIG PATHS ${HIPBLAS-COMMON_PATH})
+  find_package( hipblas-common REQUIRED CONFIG PATHS "C:/hipSDK")
```

Comment:
The C:/hipSDK should be changed to just ${ROCM_PATH} which should work on both WIN32 and linux